panic: runtime error: invalid memory address or nil pointer dereference

gitbatch: 0.4.1

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb8 pc=0x872b19]

goroutine 1 [running]:
github.com/isacikgoz/gitbatch/gui.(*Gui).renderBranches(0xc4201a6e60, 0x0, 0x0, 0xc4202320f0)
        /home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/sideviews.go:70 +0x89
github.com/isacikgoz/gitbatch/gui.(*Gui).renderSideViews(0xc4201a6e60, 0x0, 0x4, 0xc4202320f0)
        /home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/sideviews.go:50 +0x5a
github.com/isacikgoz/gitbatch/gui.(*Gui).renderMain(0xc4201a6e60, 0x0, 0x0)
        /home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/repositoriesview.go:29 +0x1ce
github.com/isacikgoz/gitbatch/gui.(*Gui).repositoryUpdated.func1(0xc420210000, 0x1, 0xc42011fb40)
        /home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/repositoriesview.go:35 +0x2a
github.com/jroimartin/gocui.(*Gui).MainLoop(0xc420210000, 0xc420210000, 0x0)
        /home/isacikgoz/.local/share/go/src/github.com/jroimartin/gocui/gui.go:377 +0x1e0
github.com/isacikgoz/gitbatch/gui.(*Gui).Run(0xc4201a6e60, 0x0, 0x0)
        /home/isacikgoz/.local/share/go/src/github.com/isacikgoz/gitbatch/gui/gui.go:140 +0x1ee
main.run(0x0, 0x0, 0x0, 0xb13646, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/isacikgoz/git/gitbatch/main.go:52 +0x122
main.main()
        /home/isacikgoz/git/gitbatch/main.go:24 +0xc2
``

--- a/gui/sideviews.go
+++ b/gui/sideviews.go
@@ -44,6 +44,9 @@ func (gui *Gui) sideCursorUp(g *gocui.Gui, v *gocui.View) error {
 }
 
 func (gui *Gui) renderSideViews(r *git.Repository) error {
+	if r == nil {
+		return nil
+	}
 	if err := gui.resetSideCursors(); err != nil {
 		return err
 	}

